mwdb/web/src/components/Shares.js

```diff
@@ -6,19 +6,29 @@ import { RefString, DateString, ObjectLink, ConfirmationModal } from "@mwdb-web/
 
 class ShareItem extends Component {
     render() {
-        const fieldStyle = {
+        let fieldStyle = {
             wordBreak: 'break-all'
         };
+        let current_object = this.props.access_reason.includes(this.props.id)
+        let uploader = this.props.access_reason.includes(this.props.group_name)
```

Comment:
Oh, that one has lots of issues.

1. Don't use `access_reason` string, it's just left for API backwards compatibility. Why do we want to use `access_reason` if all fields are decomposed into `reason_type` and `related_*` things?
2. In JS we should use camelCase for identifiers. But access_reason and group_name comes from API so we can't do anything here
3. I would add `is` to the identifier, indicating a boolean flag rather than actual object
4. It might be better to use const here.
```suggestion
        const isCurrentObject = this.props.related_object_dhash == this.props.id
        const isUploader = this.props.related_user_login == this.props.group_name
```